docs/contracts/overview/DigitalAssets.md

```diff
@@ -136,6 +136,114 @@ You can filter the `DataChanged` event with the following `dataKey`:
 
 </details>
 
+### Setting metadata for one or multiple `tokenId`s
+
+The function [`setDataBatchForTokenIds(bytes32[],bytes32[],bytes[])`](../../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md#setdatabatchfortokenids) can be used to set multiple data key-value pairs at once for one or multiple tokenIds.
+
+This function is flexible enough to enable to:
+
+- **case 1:** either set multiple data key-value pairs for the same tokenId.
+- **case 2:** set data key-value pairs for different tokenIds.
```

Comment:
```suggestion
- a single `tokenId`
- different `tokenId` elements
```

- I think there is no need to keep the `either` as the bullet points do not finish the sentence.
- I would rather create separate menus / headings in order to keep the file page more organized.

Reply:
Done created different subheadings